apps/web-mzima-client/src/app/post/post-preview/post-preview.component.html

```diff
@@ -31,7 +31,7 @@
 
   <div class="post__main">
     <div class="post__content">
-      <h3 *ngIf="post.title?.length || post.content?.length">{{ post.title || post.content }}</h3>
+      <p *ngIf="post.title?.length || post.content?.length">{{ post.title || post.content }}</p>
       <p *ngIf="post.title?.length && post.content?.length">{{ post.content }}</p>
```

Comment:
Can you explain a little more why the h3 is changed to a p here?

Reply:
This was a mistake, the change is meant to be for another file in another pr. Would revert now.